src/fragments/lib/geo/js/getting-started.mdx

```diff
@@ -7,7 +7,7 @@ Amplify Geo provides APIs and map UI components for maps and location search for
 
 Follow this guide to get started with Amplify Geo through the Amplify CLI.
 
-**Note:** If you want to use existing Amazon Location Service resources [follow this guide](/lib/geo/existing-resources) instead.
+**Note:** If you want to use existing Amazon Location Service resources [follow this guide](/lib/geo/existing-resources) instead. If you want to use Amazon Location Service APIs not directly supported by Geo, use the [escape hatch](/lib/geo/escapehatch) to access the Amazon Location Service SDK.
 
```

Comment:
I feel the want to combine them into one, but now they feel like they run together. Perhaps a different separation, like bullets? @hdworld11?

```suggestion
**Notes:** 
- If you want to use existing Amazon Location Service resources [follow this guide](/lib/geo/existing-resources) instead.
- If you want to use Amazon Location Service APIs not directly supported by Geo, use the [escape hatch](/lib/geo/escapehatch) to access the Amazon Location Service SDK.

```